packages/web/src/app/shared/local-theme-switch/local-theme-switch.component.ts

```diff
@@ -0,0 +1,25 @@
+import { CUSTOM_ELEMENTS_SCHEMA, Component, input, output } from '@angular/core';
+
+import { Theme } from 'src/app/core/services/theme.service';
+
+@Component({
+  selector: 'app-local-theme-switch',
+  standalone: true,
+  imports: [],
+  templateUrl: './local-theme-switch.component.html',
+  schemas: [CUSTOM_ELEMENTS_SCHEMA],
+})
+export class LocalThemeSwitchComponent {
+  themeChange = output<Theme>();
+  theme = input<Theme>();
```

Comment:
We might be able to use the Model input here to get the output "for free": https://angular.io/guide/model-inputs#implicit-change-events